vcd/datasource_vcd_vapp_vm.go

```diff
@@ -107,6 +107,11 @@ func datasourceVcdVAppVm() *schema.Resource {
 							Type:        schema.TypeString,
 							Description: "Mac address of network interface",
 						},
+						"adapter_type": {
+							Computed:    true,
```

Comment:
Please swap lines, we use `Type` everywhere as first

Reply:
Done.